relation_engine_server/utils/ensure_specs.py

```diff
@@ -0,0 +1,265 @@
+"""
+Ensure that all the specs in the spec/**/*.json and spec/**/*.yaml are
+present in the server, with the top-level fields of the local specs being
+a subset of the top-level fields of the server specs
```

Comment:
Just to make sure I understand what's going on -- the intention here is that the server admin or other interested party will run `ensure_all` to check that the specs they have locally match what's on the server. Is that right? Should you also be checking the collections and stored queries match?

What happens if someone who doesn't have the specs locally wants to check that the server has all the expected specs?

Reply:
(1) Correct. I hope that's clear ... at "some point" perhaps there should be more prose-y documentation on this. The collections and stored queries are in the clear I think because I don't think they have specs in the server.
(2) It should be possible, but that's not how it's set up in the startup script and in the tests.
(3) `download_specs` is triggered on the last line of the Dockerfile
(4) Validating what against what? There's code to validate the specs against meta-specs ... not sure where they're triggered from though